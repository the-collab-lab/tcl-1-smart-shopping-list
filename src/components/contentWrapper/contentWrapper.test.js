import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import ContentWrapper from './index';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter>
      <ContentWrapper>must have content</ContentWrapper>
    </BrowserRouter>,
    div
  );
  ReactDOM.unmountComponentAtNode(div);
});

it('snapshot on load', () => {
  const tree = TestRenderer.create(
    <BrowserRouter>
      <ContentWrapper>must have content</ContentWrapper>
    </BrowserRouter>
  );

  expect(tree.toJSON()).toMatchSnapshot();
});
