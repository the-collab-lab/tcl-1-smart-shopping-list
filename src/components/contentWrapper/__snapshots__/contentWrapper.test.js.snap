// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot on load 1`] = `
<section
  className="css-1xpf03y"
>
  must have content
</section>
`;
